mPower2/mPower2/UIColor+mPowerTheme.swift

```diff
@@ -86,6 +90,14 @@ extension UIColor {
     
     // MARK: Royal Purple
     
+    //    Very Light Purple (unfilled progress circle):
+    //    Royal 200
+    //    #DBD9D9
+    //    219, 217, 217
+    class var royal200: UIColor {
+        return UIColor(red: 219.0 / 255.0, green: 217.0 / 255.0, blue: 217.0 / 255.0, alpha: 1)
+    }
```

Comment:
See comment above. Where are you getting the very light color? From Stockard? It needs to be consistent throughout the app. Elsewhere, the app uses light grey for progress backgrounds on white.

Reply:
fixed